src/ReactiveUI.Fody.Helpers/ReactiveUI.Fody.Helpers.csproj

```diff
@@ -1,8 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461;uap10.0.16299;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10;MonoAndroid81;netcoreapp2.0</TargetFrameworks>
-    <AssemblyName>ReactiveUI.Fody.Helpers</AssemblyName>
```

Comment:
The default of both AssemblyName/RootNamespace is the name of the .csproj, so redundant.